src/components/Tetris/index.js

```diff
@@ -1,19 +1,36 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './style.css';
+
+// Custom hooks
+import usePlayer from '../../hooks/usePlayer';
+import useStage from '../../hooks/useStage';
+
+// Components
 import Stage from '../Stage';
 import Display from '../Display';
 import StratButton from '../StartButton';
 
 const Tetris = () => {
+  const [dropTime, setDropTime] = useState(null);
+  const [gameOver, setGameOver] = useState(false);
+
+  const [player] = usePlayer();
+  const [stage, setStage] = useStage(player);
+
+  console.log('re-render');
   return (
     <div className="tetris">
-      <Stage />
+      <Stage stage={stage} />
       <aside className="side">
-        <div className="side-show">
-          <Display text="Score" />
-          <Display text="Rows" />
-          <Display text="Level" />
-        </div>
+        {gameOver ? (
```

Comment:
it's better to put this line and this condition inside
to save the style.
```suggestion
<div className="side-show">
        {gameOver ? (
          <Display gameOver={gameOver} text="Game Over" />
        ) : (
            <Display text="Score" />
            <Display text="Rows" />
            <Display text="Level" />
        )}
 </div>
```

Reply:
Yeah, I resolve this, Thank you 👍
```js
<div className="side-show">
          {gameOver ? (
            <Display gameOver={gameOver} text="Game Over" />
          ) : (
            <>
              <Display text="Score" />
              <Display text="Rows" />
              <Display text="Level" />
            </>
          )}
        </div>
```